test.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
@neohere97 why did you add this in https://github.com/cu-ecen-aeld/assignment-1-neohere97/commit/c80f7657f88c01bc57987a568ccf06ae393e6cae and where did you find it?  This is an old script from previous semesters which shouldn't be used anymore.

Reply:
@dwalkes I found it in the assignments-base code which we had cloned. I followed the instructions on one of the github repository/coursera to add the test-basedir.sh to the base repository as test.sh.

Reply:
You mean you followed the instructions in https://github.com/cu-ecen-aeld/assignment-autotest ?  This one is already setup for you as a submodule at https://github.com/cu-ecen-aeld/aesd-assignments so there's nothing you need to do for that one.  The instructions in Coursera don't reference assignment-autotest, just aesd-assignments